README.rst

```diff
@@ -593,6 +602,18 @@ For example, if we want to change the verbose name to "Hotspot", we could write:
 
     OPENWISP_CONTROLLER_DEVICE_VERBOSE_NAME = ('Hotspot', 'Hotspots')
 
+``OPENWISP_SUBNET_HIDE_AUTOMATICALLY_GENERATED_SUBNETS_AND_IPS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
the prefix `OPENWISP_SUBNET` is not consistent with the other settings, we should use `OPENWISP_CONTROLLER`